src/rest/AllowedValuesPatternType.ts

```diff
@@ -0,0 +1,5 @@
+export enum AllowedValuesPatternType {
+  Legacy = 'legacy',
```

Comment:
Could the enum keys be lowercase? or is it a convention to use uppercase for enums?